ci-operator/templates/cluster-launch-e2e-openshift-jenkins.yaml

```diff
@@ -214,12 +215,14 @@ objects:
         mkdir -p /tmp/output
         cd /tmp/output
 
+        # we are not setting up the telemetry client like in cluster-launch-e2e.yaml since it was
```

Comment:
Keep it consistent, so that copying it easier.

Reply:
will do